05/15/2018 02:51:03 PM|ERROR|Exception on /api/v1/hbs/activation [GET]|Traceback (most recent call last):  File "C:\Python27\lib\site-packages\flask\app.py", line 1639, in full_dispatch_request    rv = self.dispatch_request()  File "C:\Python27\lib\site-packages\flask\app.py", line 1625, in dispatch_request    return self.view_functions[rule.endpoint](**req.view_args)  File "C:\Python27\lib\site-packages\flask_restful\__init__.py", line 480, in wrapper    resp = resource(*args, **kwargs)  File "C:\Python27\lib\site-packages\flask\views.py", line 84, in view    return self.dispatch_request(*args, **kwargs)  File "C:\Python27\lib\site-packages\flask_restful\__init__.py", line 595, in dispatch_request    resp = meth(*args, **kwargs)  File "C:\Python27\lib\site-packages\flask_jwt\__init__.py", line 177, in decorator    return fn(*args, **kwargs)  File "C:\Users\adir_s\Desktop\git-projects\flask\FlaskServer\Routes\ActivationWizard\__init__.py", line 33, in get    return onGetRequest(request)  File "C:\Users\adir_s\Desktop\git-projects\flask\FlaskServer\Routes\ActivationWizard\__init__.py", line 55, in onGetRequest    data = load_mapping_json_file(os.path.join(__location__, 'activation.json'))  File "C:\Users\adir_s\Desktop\git-projects\flask\FlaskServer\jsonutils.py", line 162, in load_mapping_json_file    data = byteify(json.load(data_file))  File "C:\Python27\lib\json\__init__.py", line 291, in load    **kw)  File "C:\Python27\lib\json\__init__.py", line 339, in loads    return _default_decoder.decode(s)  File "C:\Python27\lib\json\decoder.py", line 364, in decode    obj, end = self.raw_decode(s, idx=_w(s, 0).end())  File "C:\Python27\lib\json\decoder.py", line 380, in raw_decode    obj, end = self.scan_once(s, idx)ValueError: Expecting , delimiter: line 29 column 5 (char 2125)|
05/15/2018 02:51:08 PM|ERROR|Exception on /api/v1/hbs/activation [GET]|Traceback (most recent call last):  File "C:\Python27\lib\site-packages\flask\app.py", line 1639, in full_dispatch_request    rv = self.dispatch_request()  File "C:\Python27\lib\site-packages\flask\app.py", line 1625, in dispatch_request    return self.view_functions[rule.endpoint](**req.view_args)  File "C:\Python27\lib\site-packages\flask_restful\__init__.py", line 480, in wrapper    resp = resource(*args, **kwargs)  File "C:\Python27\lib\site-packages\flask\views.py", line 84, in view    return self.dispatch_request(*args, **kwargs)  File "C:\Python27\lib\site-packages\flask_restful\__init__.py", line 595, in dispatch_request    resp = meth(*args, **kwargs)  File "C:\Python27\lib\site-packages\flask_jwt\__init__.py", line 177, in decorator    return fn(*args, **kwargs)  File "C:\Users\adir_s\Desktop\git-projects\flask\FlaskServer\Routes\ActivationWizard\__init__.py", line 33, in get    return onGetRequest(request)  File "C:\Users\adir_s\Desktop\git-projects\flask\FlaskServer\Routes\ActivationWizard\__init__.py", line 55, in onGetRequest    data = load_mapping_json_file(os.path.join(__location__, 'activation.json'))  File "C:\Users\adir_s\Desktop\git-projects\flask\FlaskServer\jsonutils.py", line 162, in load_mapping_json_file    data = byteify(json.load(data_file))  File "C:\Python27\lib\json\__init__.py", line 291, in load    **kw)  File "C:\Python27\lib\json\__init__.py", line 339, in loads    return _default_decoder.decode(s)  File "C:\Python27\lib\json\decoder.py", line 364, in decode    obj, end = self.raw_decode(s, idx=_w(s, 0).end())  File "C:\Python27\lib\json\decoder.py", line 380, in raw_decode    obj, end = self.scan_once(s, idx)ValueError: Expecting , delimiter: line 29 column 5 (char 2125)|
05/16/2018 08:56:32 AM|ERROR|Exception on /api/v1/hbs/activation/link [GET]|Traceback (most recent call last):  File "C:\Python27\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request    rv = self.dispatch_request()  File "C:\Python27\lib\site-packages\flask\app.py", line 1598, in dispatch_request    return self.view_functions[rule.endpoint](**req.view_args)  File "C:\Python27\lib\site-packages\flask_restful\__init__.py", line 480, in wrapper    resp = resource(*args, **kwargs)  File "C:\Python27\lib\site-packages\flask\views.py", line 84, in view    return self.dispatch_request(*args, **kwargs)  File "C:\Python27\lib\site-packages\flask_restful\__init__.py", line 595, in dispatch_request    resp = meth(*args, **kwargs)  File "C:\Python27\lib\site-packages\flask_jwt\__init__.py", line 177, in decorator    return fn(*args, **kwargs)  File "C:\RadWin\GIT\flask\FlaskServer\Routes\ActivationWizard\__init__.py", line 35, in get    if name == ACTIVATION_WIZARD_LINK_URL:NameError: global name 'ACTIVATION_WIZARD_LINK_URL' is not defined|
05/16/2018 08:57:21 AM|ERROR|Exception on /api/v1/hbs/activation/link [GET]|Traceback (most recent call last):  File "C:\Python27\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request    rv = self.dispatch_request()  File "C:\Python27\lib\site-packages\flask\app.py", line 1598, in dispatch_request    return self.view_functions[rule.endpoint](**req.view_args)  File "C:\Python27\lib\site-packages\flask_restful\__init__.py", line 480, in wrapper    resp = resource(*args, **kwargs)  File "C:\Python27\lib\site-packages\flask\views.py", line 84, in view    return self.dispatch_request(*args, **kwargs)  File "C:\Python27\lib\site-packages\flask_restful\__init__.py", line 595, in dispatch_request    resp = meth(*args, **kwargs)  File "C:\Python27\lib\site-packages\flask_jwt\__init__.py", line 177, in decorator    return fn(*args, **kwargs)  File "C:\RadWin\GIT\flask\FlaskServer\Routes\ActivationWizard\__init__.py", line 35, in get    if name == ACTIVATION_WIZARD_LINK_URL:NameError: global name 'ACTIVATION_WIZARD_LINK_URL' is not defined|
05/16/2018 02:34:21 PM|ERROR|Exception on /api/v1/hbs/activation/radio [GET]|Traceback (most recent call last):  File "C:\Python27\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request    rv = self.dispatch_request()  File "C:\Python27\lib\site-packages\flask\app.py", line 1598, in dispatch_request    return self.view_functions[rule.endpoint](**req.view_args)  File "C:\Python27\lib\site-packages\flask_restful\__init__.py", line 480, in wrapper    resp = resource(*args, **kwargs)  File "C:\Python27\lib\site-packages\flask\views.py", line 84, in view    return self.dispatch_request(*args, **kwargs)  File "C:\Python27\lib\site-packages\flask_restful\__init__.py", line 595, in dispatch_request    resp = meth(*args, **kwargs)  File "C:\Python27\lib\site-packages\flask_jwt\__init__.py", line 177, in decorator    return fn(*args, **kwargs)  File "C:\RadWin\GIT\flask\FlaskServer\Routes\ActivationWizard\__init__.py", line 46, in get    return onRadioGetRequest(request)  File "C:\RadWin\GIT\flask\FlaskServer\Routes\ActivationWizard\__init__.py", line 108, in onRadioGetRequest    data = load_mapping_json_file(os.path.join(__location__, 'activation.json'))['radio']  File "C:\RadWin\GIT\flask\FlaskServer\jsonutils.py", line 162, in load_mapping_json_file    data = byteify(json.load(data_file))  File "C:\Python27\lib\json\__init__.py", line 291, in load    **kw)  File "C:\Python27\lib\json\__init__.py", line 339, in loads    return _default_decoder.decode(s)  File "C:\Python27\lib\json\decoder.py", line 364, in decode    obj, end = self.raw_decode(s, idx=_w(s, 0).end())  File "C:\Python27\lib\json\decoder.py", line 380, in raw_decode    obj, end = self.scan_once(s, idx)ValueError: Expecting property name: line 18 column 5 (char 1706)|
05/16/2018 02:34:54 PM|ERROR|Exception on /api/v1/hbs/activation/radio [GET]|Traceback (most recent call last):  File "C:\Python27\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request    rv = self.dispatch_request()  File "C:\Python27\lib\site-packages\flask\app.py", line 1598, in dispatch_request    return self.view_functions[rule.endpoint](**req.view_args)  File "C:\Python27\lib\site-packages\flask_restful\__init__.py", line 480, in wrapper    resp = resource(*args, **kwargs)  File "C:\Python27\lib\site-packages\flask\views.py", line 84, in view    return self.dispatch_request(*args, **kwargs)  File "C:\Python27\lib\site-packages\flask_restful\__init__.py", line 595, in dispatch_request    resp = meth(*args, **kwargs)  File "C:\Python27\lib\site-packages\flask_jwt\__init__.py", line 177, in decorator    return fn(*args, **kwargs)  File "C:\RadWin\GIT\flask\FlaskServer\Routes\ActivationWizard\__init__.py", line 46, in get    return onRadioGetRequest(request)  File "C:\RadWin\GIT\flask\FlaskServer\Routes\ActivationWizard\__init__.py", line 108, in onRadioGetRequest    data = load_mapping_json_file(os.path.join(__location__, 'activation.json'))['radio']  File "C:\RadWin\GIT\flask\FlaskServer\jsonutils.py", line 162, in load_mapping_json_file    data = byteify(json.load(data_file))  File "C:\Python27\lib\json\__init__.py", line 291, in load    **kw)  File "C:\Python27\lib\json\__init__.py", line 339, in loads    return _default_decoder.decode(s)  File "C:\Python27\lib\json\decoder.py", line 364, in decode    obj, end = self.raw_decode(s, idx=_w(s, 0).end())  File "C:\Python27\lib\json\decoder.py", line 380, in raw_decode    obj, end = self.scan_once(s, idx)ValueError: Expecting property name: line 18 column 5 (char 1706)|
